templates/charms-architecture.html

```diff
@@ -0,0 +1,243 @@
+{% extends 'how-juju-works.html' %}
+
+{% block title %}Charms architecture{% endblock %}
+{% block meta_keywords %}Software operators{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1nMN5UlwGqI4C4GmYcKy4yRzd9ANUmsVyB1ZcKY7Wpro/edit#heading=h.4tjfso5hx6tw{% endblock meta_copydoc %}
+{% block page_class %}is-paper{% endblock %}
+
+{% set selected_tab='charms-architecture' %}
+
+{% block how_juju_works_content %}
+<section class="p-section">
+  <div class="row--50-50">
+    <div class="col">
+      <h1 class="p-heading--2">What is a charm?</h1>
+    </div>
+    <div class="col">
+      <p class="p-heading--5">A charm is software that wraps an application and that contains all of the instructions necessary for deploying, configuring, operating an application on any cloud using Juju.</p>
+      <p>A charm generally contains the Juju unit agent, the operations (charm) code and the application itself (the workload). Each application has a number of deployed "units" and includes information about the services they offer or require. Charms scale with the application and the operations code is always right next to the workload it is driving.</p>
+      <p>Charms include deterministic logic that specifies what happens when specific events occur. Events can be triggered by an administrator (e.g. through the CLI), by other charms or the external environment.</p>
+      <p>The administrator interacts with a client (CLI, terraform or Jimm), which talks to the controller. The controller talks to the agent in the charm, which in turns invokes the charm code to manipulate the application.</p>
+      <p>Charms are currently of two kinds, depending on the target deployment substrate:</p>
+      <ol>
```

Comment:
Not sure what the designers have to say here, but I believe or current default lists should use the "divided" (`p-list--divided`) style, not plain list like this:

<img width="688" alt="image" src="https://github.com/canonical/juju.is/assets/83575/784872ed-de17-4bb9-92b5-81b891e0c04f">

https://vanillaframework.io/docs/patterns/lists#ordered-lists-with-horizontal-divider

This may apply to other places where plain `ol` or `ul` are used.

Reply:
Good shout, I'll use `p-list--divided` style for these and update the design